modules/upi-provisioning-new-hosts-in-a-upi-cluster.adoc

```diff
@@ -20,45 +20,156 @@ To provision bare-metal hosts to the cluster by using the BMO, you must set the
 
 .Procedure
 
-. Create the `Secret` CR and the `BareMetalHost` CR.
-
-.. Save the following YAML in the `bmh.yaml` file:
+. Create a configuration file for the bare-metal node. Depending on whether you are using a static configuration or a DHCP server, use one of the following example `bmh.yaml` files, replacing values in the YAML to match your environment:
++
+.Static configuration `bmh.yaml`
 +
 [source,yaml]
 ----
 ---
 apiVersion: v1
 kind: Secret
 metadata:
-  name: worker1-bmc
+  name: openshift-worker-<num>-network-config-secret # <1>
+  namespace: openshift-machine-api
+type: Opaque
+stringData:
+ nmstate: | # <2>
+  interfaces: # <3>
+  - name: <nic1_name> # <4>
+    type: ethernet
+    state: up
+    ipv4:
+      address:
+      - ip: <ip_address> # <5>
+        prefix-length: 24
+      enabled: true
+  dns-resolver:
+    config:
+      server:
+      - <dns_ip_address> # <6>
+  routes:
+    config:
+    - destination: 0.0.0.0/0
+      next-hop-address: <next_hop_ip_address> # <7>
+      next-hop-interface: <next_hop_nic1_name> # <8>
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: openshift-worker-<num>-bmc-secret # <1>
   namespace: openshift-machine-api
 type: Opaque
 data:
-  username: <base64_of_uid>
-  password: <base64_of_pwd>
+  username: <base64_of_uid> # <9>
+  password: <base64_of_pwd> # <9>
```

Comment:
Reuse of callouts is not supported on docs.redhat.com. I see a few in this PR - won't comment them all but please go through and make sure all are unique.